signature-specification.md

```diff
@@ -1,243 +1,118 @@
+    }
+  }
 }
 ```
 
-```json
+```jsonc
 {
-   "mediaType": "sbom/example",
-   "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
-   "size": 32654
+  "subject": {
+    "mediaType": "sbom/example",
+    "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
+    "size": 32654
+  }
 }
 ```
 
 ### Signed Attributes
 
-Notary v2 requires the signature envelope to support the following signed attributes:
-
-- **Creation time**: The time at which the signature was generated.
-  Its value should be numeric representing the number of seconds (not milliseconds) since Epoch.
-- **Expiration time**: The time after which signature shouldn't be considered valid.
-  Its value should be numeric representing the number of seconds since Epoch.
-  This is an OPTIONAL attribute.
+Signed attributes/claims are additional metadata apart from the payload, which are required to support the signature verification process.
 
-### Unsigned Attributes
+- Any claims MUST NOT be stored/appended in the payload itself, as the payload is only parsed and processed once the signature has been verified (signature is valid, and from a trusted key) and trust is established.
+- Specific claims can be either required or optional.
+- Claims that if present, MUST be processed by a verifier MUST be marked as critical.
+- Claims which are informational and do not influence signature verification MUST NOT be marked critical.
 
-- **Certificates**: The ordered collection of X.509 certificates with a signing certificate as the first entry.
-- **Timestamp**:  The time stamp token generated for a given signature.
-  Only [RFC3161](https://datatracker.ietf.org/doc/html/rfc3161#section-2.4.2) compliant *TimeStampToken* are supported.
-  This is an OPTIONAL attribute.
+Notary v2 requires the signature envelope to support the following signed attributes/claims.
 
-### Supported Signature Envelopes
+#### Standard attributes
 
-#### JWS JSON Serialization
+- **Signing time**: The time at which the signature was generated. Though this claim is signed by the signing key, it’s considered unauthenticated as a signer can modify local time and manipulate this claim. More details [here](#signing-time).
+- **Expiry** (critical): An OPTIONAL claim that provides a “best by use” time for the artifact, as defined by the signer. More details [here](#expiry).
+- **Content type** (critical): A REQUIRED clain that indicates the content type of the payload. Notary currently a payload that contains the OCI descriptor of a subject manifest as the payload, the supported value is `application/vnd.cncf.notary.payload.v1+json`, other payload types MAY be supported in future.
+- **Client identifier**: The version of a client (e.g. Notation) that produced the signature. This is an OPTIONAL claim. It uses the following format `{client}/{version}` e.g. “notation/1.0.0”. This claim in intended to be used for diagnostic and troubleshooting purposes.
```

Comment:
Curious if `v` is allowed in `{version}` like `notation/v1.0.0`?

Reply:
Should this be a signed attribute? Since it is for debugging purposes only, can it be unsigned?

Reply:
> Curious if `v` is allowed in `{version}` like `notation/v1.0.0`?

Yes, I'll update this section that its an opaque string, and only make a suggestion about the format to be used (formatting is not validated).

Reply:
> Should this be a signed attribute? Since it is for debugging purposes only, can it be unsigned?

Changed this to an unsigned attribute, as this attribute is informational, and cannot be a signed attribute when a `envelope-generator` plugin is used.